Config.cmake/config_test.h.cmake

```diff
@@ -25,10 +25,8 @@
 /* PATHs to temporary test directory */
 #define FMU_TEMPORARY_TEST_DIR "@FMU_TEMPORARY_TEST_DIR@"
 
-#define COMPRESS_DUMMY_FILE_PATH_SRC "@COMPRESS_DUMMY_FILE_PATH_SRC@"
-#define COMPRESS_DUMMY_FILE_PATH_DIST "@COMPRESS_DUMMY_FILE_PATH_DIST@"
-#define UNCOMPRESSED_DUMMY_FILE_PATH_SRC "@UNCOMPRESSED_DUMMY_FILE_PATH_SRC@"
-#define UNCOMPRESSED_DUMMY_FOLDER_PATH_DIST "@UNCOMPRESSED_DUMMY_FOLDER_PATH_DIST@"
+#define TEST_OUTPUT_FOLDER "@TEST_OUTPUT_FOLDER@"
+#define FMIL_TEST_DIR "@FMIL_TEST_DIR@"
```

Comment:
Moving code from cmake to the .c/.cpp test files. It's easier to build the paths in C and doesn't clutter the cmake code far from where it's used.